strings/base_implements.h

```diff
@@ -782,6 +782,12 @@ namespace winrt::impl
     template <>
     struct WINRT_IMPL_EMPTY_BASES root_implements_composing_outer<true>
     {
+        template <typename Qi>
+        auto as() const noexcept
```

Comment:
This shouldn't be added, see #1363

Reply:
[sigh] This is needed to compile compositing types that pass through a consume_ method.  I either need this or I revert back to `try_as_with_reason`.

Reply:
Ah, I didn't see it was needed for that. Let's keep it in then.

Reply:
Reverted the commit.  Back to `try_as_with_reason` :)

Reply:
I meant keeping this `as` but this works too haha